joinmarket-webui/joinmarket-webui
# dev: use joinmarket-webui-dev-standalone in regtest setup

Changes to regtest environment:
- Use `joinmarket-webui/joinmarket-webui-dev-standalone` for second jm container
  - This will expose a UI on port `29080` - with basic auth enabled (user and password `joinmarket`). helpful for regression tests
- Use `--docker-install` flag for `latest` setup for first jm container
  - Deliberately not using `standalone` as base because everyone should be able to run any server version quickly
- Remove image based on `btcpayserver/joinmarket:v0.9.3`
